wpeverest/user-registration
# Enhance - Notice display method to display review and survey notice

### All Submissions:






### Changes proposed in this Pull Request:
This PR contains the following changes :
1. Previously the review notice was displayed after 30 days of plugin activation which has been changed to 14 days.
2. A new notice i.e survey notice has been added which will be displayed if the user has activated plugin for more than 10 days and activated the license.

### How to test the changes in this Pull Request:

1. Verify if the time frame for displaying the review notice has been changed.
2. Verify if the survey notice design and contents are valid.
3. Verify if the review and survey notice both are displayed as per the conditions.
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/85762481/128803865-40a8199c-a0c9-4218-adf7-829be76e3da9.png)

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Notice display method to display review and survey notice.
